experimental/dispatch_profiler/manifest.py

```diff
@@ -200,8 +200,8 @@ def append(self, dispatch_collection_list):
   def initialize(self):
     """Initialize the mainfest object by generating dispatches for supported operations."""
     self.append(CudaMatmulGenerator(self.args).generate())
-    self.append(CudaSplitKMatmulGenerator(self.args).generate())
-    self.append(CudaBatchMatmulGenerator(self.args).generate())
+    #self.append(CudaSplitKMatmulGenerator(self.args).generate())
```

Comment:
Similar as above, elide or document with a comment what this might be used for in future

Reply:
Sam as above. Just focusing on BF16 matmul before the PR is ready to merge.